buildflags/buildflags.gni

```diff
@@ -33,4 +33,7 @@ declare_args() {
 
   # Enable Chrome extensions support.
   enable_electron_extensions = false
+
+  # Enable Spellchecker support
+  enable_builtin_spellchecker = true
```

Comment:
@MarshallOfSound Looks like it is enabled by default but PR body says it will be disabled by default?

Reply:
I enabled it to test full CI

Reply:
New plan, I want to turn this on for Realz in this PR 👍 

It's still behind a flag for easy removal / disable by other folks.